integration.json

```diff
@@ -3,9 +3,9 @@
     "paketocommunity/builder-ubi-buildpackless-base:latest",
     "paketobuildpacks/builder-jammy-buildpackless-base"
   ],
-  "ubi-nodejs-extension": "github.com/paketo-community/ubi-nodejs-extension",
-  "node-engine": "github.com/paketo-buildpacks/node-engine",
-  "yarn": "github.com/paketo-buildpacks/yarn",
-  "yarn-install": "github.com/paketo-buildpacks/yarn-install",
-  "watchexec": "github.com/paketo-buildpacks/watchexec"
+  "ubi-nodejs-extension": "index.docker.io/paketocommunity/ubi-nodejs-extension",
+  "node-engine": "index.docker.io/paketobuildpacks/node-engine",
+  "yarn": "index.docker.io/paketobuildpacks/yarn",
+  "yarn-install": "index.docker.io/paketobuildpacks/yarn-install",
+  "watchexec": "index.docker.io/paketobuildpacks/watchexec"
```

Comment:
@pacostas did we need to change anything other than how we get watchexec? Since its a temporary work around, I'm thinking we should limit the change to the minimum required.

Reply:
I agree, i'll changed that ASAP